lib/types/beacon_chain/withdrawal.ex

```diff
@@ -20,4 +21,14 @@ defmodule Types.Withdrawal do
           address: Types.hash32(),
           amount: Types.gwei()
         }
+
+  @impl LambdaEthereumConsensus.Container
+  def schema do
+    [
+      {:index, TypeAliases.withdrawal_index()},
+      {:validator_index, TypeAliases.validator_index()},
+      {:address, TypeAliases.hash32()},
```

Comment:
This one should be a `TypeAliases.ExecutionAddress`, which is a `bytes20`, not a `hash32`, which is a `bytes32`.